pkg/server/chaosd/network.go

```diff
@@ -511,3 +532,51 @@ func (s *Server) recoverEtcHosts(attack *core.NetworkCommand, uid string) error
 	}
 	return nil
 }
+
+func (s *Server) recoverNICDown(attack *core.NetworkCommand) error {
+	NICUpCommand := fmt.Sprintf("ifconfig %s %s up", attack.Device, attack.IPAddress)
+
+	recoverCmd := exec.Command("bash", "-c", NICUpCommand)
+	stdout, err := recoverCmd.CombinedOutput()
+	if err != nil {
+		log.Error(recoverCmd.String()+string(stdout), zap.Error(err))
```

Comment:
We can delete this log. refer to https://github.com/chaos-mesh/rfcs/blob/main/text/2021-12-09-logging.md#relations-between-error-and-logging

Reply:
I removed these `log.Error`, PTAL : )